spec/migrations/20171106143927_downcase_resource_group_ems_ref_for_azure_spec.rb

```diff
@@ -0,0 +1,31 @@
+require_migration
+
+describe DowncaseResourceGroupEmsRefForAzure do
+  let(:resource_group_stub) { migration_stub :ResourceGroup }
+
+  let!(:uppercase_stub) do
+    resource_group_stub.create!(:name => 'UPPER', :ems_ref => '/subscriptions/xyz/resourceGroups/UPPER')
+  end
+
+  let!(:mixedcase_stub) do
+    resource_group_stub.create!(:name => 'MiXeD', :ems_ref => '/subscriptions/xyz/resourceGroups/MiXeD')
+  end
+
+  let!(:lowercase_stub) do
+    resource_group_stub.create!(:name => 'lower', :ems_ref => '/subscriptions/xyz/resourceGroups/lower')
+  end
+
+  migration_context :up do
+    it 'Downcases the resource group ems_ref column' do
+      migrate
+      expect(uppercase_stub.ems_ref).to eql('/subscriptions/xyz/resourcegroups/upper')
+      expect(uppercase_stub.name).to eql('UPPER')
+
+      expect(mixedcase_stub.ems_ref).to eql('/subscriptions/xyz/resourcegroups/mixed')
+      expect(mixedcase_stub.name).to eql('MiXeD')
+
+      expect(lowercase_stub.ems_ref).to eql('/subscriptions/xyz/resourcegroups/lower')
+      expect(lowercase_stub.ems_ref).to eql('lower')
```

Comment:
should this be `expect(lowercase_stub.name).to eql('lower')`

Reply:
Yep, too much copy-pasta.